src/View/AbstractDirectoryView.vala

```diff
@@ -2212,6 +2242,10 @@ namespace Files {
 
                 if (in_trash) {
                     if (clipboard != null && clipboard.has_cutted_file (null)) {
+                        paste_menuitem.add (new Granite.AccelLabel (
+                            _("Paste into Trash"),
+                            "<Ctrl>v"
+                        ));
```

Comment:
I don't see where this is used. It doesn't look like we get these menuitems when right clicking in Trash or right clicking on the Trash bookmark item

Reply:
Ah okay I understand. In that case, I'm not sure we need to add a new string to be localized here. I agree this is probably not a typical workflow and when it's in the context menu I don't think there's any ambiguity about what "Paste" will do since it's the same as any other folder

Reply:
Done